templates/common/infra/bicep/core/database/cosmos/mongo/cosmos-mongo-cluster.bicep

```diff
@@ -0,0 +1,62 @@
+metadata description = 'Azure Cosmos DB MongoDB vCore cluster'
+@maxLength(40)
+param name string
+param location string = resourceGroup().location
+param tags object = {}
+
+param administratorLogin string
+param sku string
+param storage int
+param nodeCount int
+@secure()
+param administratorLoginPassword string
+param highAvailabilityMode bool = false
+param allowAzureIPsFirewall bool = false
+param allowAllIPsFirewall bool = false
+param allowedSingleIPs array = []

```

Comment:
Alpha asc sort these after the name, location, tags params

Reply:
On it